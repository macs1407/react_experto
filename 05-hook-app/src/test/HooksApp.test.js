import React from 'react';
import '@testing-library/jest-dom';
import {shallow} from 'enzyme';
import HooksApp from '../HooksApp';

describe('Pruebas con HooksApp', () => {
    test('Debe de levantar HooksApp ', () => {
        const wraper = shallow(<HooksApp />);
        expect(wraper).toMatchSnapshot();
    });    
})
