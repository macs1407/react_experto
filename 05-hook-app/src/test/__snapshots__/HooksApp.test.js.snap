// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con HooksApp Debe de levantar HooksApp  1`] = `
<h1>
  Hola mundo
</h1>
`;
